### PR TITLE
show prolong box in ios

### PR DESCRIPTION
**Pull request by ovbm**
can be merged if ios review of 2.0.2 goes through

### CODE REVIEW

**Comment by tpreusse** [comment]
IMO let's wait two weeks or so until most people have updated. Or to be fully proper: only allow it for 2.0.2+

**Comment by ovbm** [comment]
yeah, I'll change it to only allow it on 2.0.2

**Comment by ovbm** [comment]
@tpreusse ready for review

File: components/Frame/ProlongBox.js [hunk 1]
```diff
@@ -40,10 +40,10 @@ const dayFormat = timeFormat('%d. %B %Y')
 const ProlongBox = ({ t, prolongBeforeDate, membership }) => {
   const [colorScheme] = useColorContext()
   const router = useRouter()
-  const { inNativeIOSApp } = useInNativeApp()
+  const { inNativeAppWithStripeCompatibility } = useInNativeApp()
 
   if (
-    inNativeIOSApp ||
+    inNativeAppWithStripeCompatibility ||
```
Comment (tpreusse) [inline components/Frame/ProlongBox.js:46]:
```suggestion
    (inNativeIOSApp && !inNativeAppWithStripeCompatibility) ||
```

**Review by tpreusse — COMMENTED** [review]
One logic mistake. Otherwise looks good.

Not sure if we should cache `inNativeAppWithStripeCompatibility` in a `useMemo`. Now it is computed every time `useInNativeApp` is used.

**Comment by ovbm** [comment]
I can also remove it from the hook and compute it directly in the prolong box

**Comment by ovbm** [comment]
Done, tested in safari with setting userAgent to RepublikApp/2.0.2 vs  RepublikApp/2.0.1.

**Review by tpreusse — APPROVED** [review]

**Pull request merged**